OPAL/tac/src/main/scala/org/opalj/tac/cg/android/AndroidICCAnalysis.scala

```diff
@@ -1338,6 +1373,18 @@ class IntentFilter(var receiver: ClassFile, var componentType: String) {
         clone
     }
 
+    def findString(tacCode: TACode[TACMethodParameter, DUVar[ValueInformation]], defSite: Int): Option[String] = {
```

Comment:
This method is duplicated, please refactor to a single instance accessible from all places where you need it.